src/til/ut_til/til.unit.tests.vcxproj

```diff
@@ -11,6 +11,7 @@
   <Import Project="$(SolutionDir)src\common.build.pre.props" />
   <ItemGroup>
     <ClCompile Include="BitmapTests.cpp" />
+    <ClCompile Include="OperatorTests.cpp" />

```

Comment:
Oh man I just realize the `sources`	have probably gotten out of date since we started using `til`, haven't they -.-

Reply:
Welp, I fixed `sources` at least for this project in the next revision.